web-app/src/services/API/internalAPI/axiosInstance.ts

```diff
@@ -36,7 +35,8 @@ const handleResponseErrors = (response: AxiosResponse) => {
       break;
     case 401:
       if (currentlyLoggedIn) {
-        // TODO: remove here cookie
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+        response.headers['set-cookie'] = undefined;
```

Comment:
Here You should remove a cookie from browser
https://github.com/cmp-cc/vue-cookies#api